ln/lnd.go

```diff
@@ -47,14 +47,14 @@ type LNDclient struct {
 }
 
 // AddInvoice generates an invoice with the given price and memo.
-func (c LNDclient) AddInvoice(value int64, memo string, descriptionHash []byte) (Invoice, error) {
+func (c LNDclient) AddInvoice(msats int64, memo string, descriptionHash []byte) (Invoice, error) {
 	result := Invoice{}
 
-	stdOutLogger.Printf("Adding invoice: memo=%s value=%v", memo, value)
+	stdOutLogger.Printf("Adding invoice: memo=%s msats=%v", memo, msats)
 	invoice := lnrpc.Invoice{
 		Memo:            memo,
 		DescriptionHash: descriptionHash,
-		Value:           value,
+		ValueMsat:       msats,
```

Comment:
Ensures that the same value is returned to the client as was provided. Easily reproduced with the current version e.g. via Phoenix when you want to donate 1 EUR since the donate value is not divisible by 1000.